Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -43,6 +43,7 @@ public static string GetEstablishmentsPluralName(GT groupType, eTextCase textCas
                 case GT.MultiacademyTrust:
                 case GT.SingleacademyTrust:
                 case GT.SchoolSponsor:
+                case GT.SecureSingleAcademyTrust:
```

Comment:
This collection contains other group types - should it be updated to include Secure SAT?

https://github.com/DFE-Digital/get-information-about-schools/blob/secure_16-19_groups/Web/Edubase.Web.UI/Areas/Groups/Models/GroupDetailViewModel.cs#L26-L35

```csharp
        private Dictionary<int, string> _groupTypes2Name = new Dictionary<int, string>
        {
            [(int)GT.ChildrensCentresCollaboration] = "children's centre collaboration",
            [(int)GT.ChildrensCentresGroup] = "children's centre group",
            [(int)GT.Federation] = "federation",
            [(int)GT.MultiacademyTrust] = "multi-academy trust",
            [(int)GT.SchoolSponsor] = "school sponsor",
            [(int)GT.SingleacademyTrust] = "single-academy trust",
            [(int)GT.Trust] = "trust"
        };
```

Reply:
added new value but I can't see any reference to the property that uses it.
can we resolve this based on adding "secure single-academy trust" as a value?

Reply:
Going to mark this as resolved because it looks like this is only used by the GroupTypeNameForClosureLabel property.
This property is only a text description used in Details.cshtml and the value is consistent with the other values now so this should be ok now.